src/feature.rs

```diff
@@ -170,7 +170,7 @@ fn expand_scenario(
             // This is done to differentiate `Hash`es of
             // scenario outlines with the same examples.
             expanded.position = position;
-            expanded.position.line += id + 1;
```

Comment:
This change is done, because original intention was to point expanded `Scenario` line onto exact cell from `Examples`. For this we need to add 2 lines: one for `Examples:` line itself and the header of the table.